modules/stampcollection.py

```diff
@@ -11,6 +11,9 @@
 
 
 vote_strengths_per_emoji = {
+ "aisafetyinfo": 1,
+ "stampy": 1,
+ "stampyog": 1,
  "stamp": 1,
  "goldstamp": 5
```

Comment:
so all except `42`? 😅 

<img width="310" alt="Screenshot 2023-02-24 at 15 33 56" src="https://user-images.githubusercontent.com/1087670/221204585-0b6d81a1-e257-4e84-9205-a4b44c4a3d65.png">

Reply:
Yeah, I mean, that's not a stamp?

Reply:
![image](https://user-images.githubusercontent.com/931092/221208640-0cbb2eef-c941-4573-9c7a-9da548d0a8fa.jpeg)